src/components/Button/Button.jsx

```diff
@@ -7,7 +7,10 @@ import styled from 'styled-components';
 const StyledButton = styled(CarbonButton)`
   &&& {
     justify-content: flex-start;
-
+    .bx--loading {
```

Comment:
If the consumer of this component has configured their codebase to use a different prefix than the default `bx`, these styles will not apply. Instead of the hardcoded `.bx`, we can replace it with the prefix variable:

```
import {settings} from carbon-components;

const {prefix} = settings;


... then in the style declaration

.${prefix}--loading {

}
```

Reply:
Done. Thanks!